python/lsst/ctrl/bps/bps_utils.py

```diff
@@ -78,6 +81,34 @@ def chdir(path):
         os.chdir(cur_dir)
 
 
+def get_mem_peaks():
```

Comment:
I agree, it's not. To make things even funnier, originally it was located in `timer.py` because I made `time_this` to report these values. However, I came to the conclusion that it may be rather misleading (after all it's a global value for the calling process, not just a snippet one used `time_this` on) and I move it to *ctrl_bps*.

Anyway, as you suggested, I'll place these functions in a new module `lsst.utils.usage`.  (I think I'd prefer `lsst.utils.usage` over `lsst.utils.memory`. This way the package could possibly be a good location for future functions related to measuring consumption of other resources).